test/strategies/elkilpstrategy.js

```diff
@@ -0,0 +1,353 @@
+const {expect} = require("chai")
+const {ethers, run} = require("hardhat")
+const {BigNumber} = ethers
+
+const MAX_UINT = "115792089237316195423570985008687907853269984665640564039457584007913129639935";
```

Comment:
seems unnecessary since we have `ethers.constants.MaxUint256`

Reply:
Idd , done